TACC/tup-ui
# Add plugin for rapid generation of staff pages - require only certain fields

## Overview & Changes

Require only names, title, and e-mail.

## Related

- changes #269

## Testing

1. Add Staff Profile plugin.
2. Verify only these fields are required.
	- First Name
	- Last Name
	- Title
	- Email

## UI

![required fields](https://github.com/TACC/tup-ui/assets/62723358/35aa2cae-c5a6-4806-aada-f1bfeb6bd586)
